R/tmb.R

```diff
@@ -233,14 +272,15 @@ h_mmrm_tmb_data <- function(formula_parts,
 #' @param tmb_data (`mmrm_tmb_data`)\cr produced by [h_mmrm_tmb_data()].
 #' @param start (`numeric` or `NULL`)\cr optional start values for variance
 #'   parameters.
-#'
+#' @param n_group `int` number of groups. Default set to 1L.
```

Comment:
```suggestion
#' @param n_groups (`int`)\cr number of groups.
```
(conventions, incl. we don't mention defaults extra here)

also good to be consistent with the plural s here for the argument name, please adapt inside